LEGO1/lego/legoomni/src/build/legocarbuildpresenter.cpp

```diff
@@ -65,11 +71,103 @@ void LegoCarBuildAnimPresenter::ReadyTickle()
 	// TODO
 }
 
-// STUB: LEGO1 0x100789e0
-// STUB: BETA10 0x10070cdd
+// FUNCTION: LEGO1 0x100789e0
+// FUNCTION: BETA10 0x10070cdd
```

Comment:
The code is mostly, but not exactly identical on BETA10. The match on BETA10 can be improved by adding `int dummy1, dummy2;` somewhere and disabling lines 90-92 as well as line 169. The highest I managed to get to was around 70 %.